examples/provider_manufacturer/README.md

```
@@ -8,8 +8,8 @@ Get a quote:
 $ pc supply quote cylinder#1
 ...
 INFO:  The following quotes are received:
-INFO:  		/pub/examples/partcad/provider_manufacturer:my3dPrinter: 12345: $100.00
-INFO:  			/pub/examples/partcad/provider_manufacturer:cylinder#1
+INFO:  		//pub/examples/partcad/provider_manufacturer:my3dPrinter: 12345: $100.00
```

Markdownlint (0.37.0)

Column: 8
Hard tabs

(MD010, no-hard-tabs)